library/components/TTooltip.vue

```diff
@@ -38,14 +38,19 @@ export default {
 			type: String,
 			default: "top"
 		},
+		isVisible: { // To control state from a component.
+			type: Boolean,
+			default: false,
+		}
```

Comment:
@sareg0 You were right about controlling it from component, but at the same time we need internal state as well for pages 😅, I am thinking about having both, I have seen this approach in other CLs as well, let me know if you have thoughts on it.

Reply:
suggestion: split the big number and tooltip fixes

Reply:
hi @sareg0 splitted the tooltip [PR](https://github.com/topcoat-data/topcoat-public/pull/134)